src/sql/workbench/contrib/jobManagement/browser/jobActions.ts

```diff
@@ -47,17 +47,8 @@ export class JobsRefreshAction extends Action {
 		super(JobsRefreshAction.ID, JobsRefreshAction.LABEL, 'refreshIcon');
 	}
 
-	public run(context: IJobActionInfo): Promise<boolean> {
-		return new Promise<boolean>((resolve, reject) => {
-			if (context) {
-				if (context.component) {
-					context.component.refreshJobs();
-				}
-				resolve(true);
-			} else {
-				reject(false);
-			}
-		});
+	public async run(context: IJobActionInfo): Promise<void> {
+		context.component.refreshJobs();
```

Comment:
this still needs the conditional for `context` and `context.component` or will throw and error

Reply:
Why would we ever not have a context with a component? That seems like it'd be a bug anyways since it means that we wouldn't be doing anything. 

I'll fix it to avoid changing behavior, but that check seems odd to me to begin with.